third_party/gtest-1.8.0/src/gtest.cc

```diff
@@ -3212,6 +3212,8 @@ void PrettyUnitTestResultPrinter::OnTestIterationEnd(const UnitTest& unit_test,
     PrintFailedTests(unit_test);
     printf("\n%2d FAILED %s\n", num_failures,
                         num_failures == 1 ? "TEST" : "TESTS");
+  } else {
+	  ColoredPrintf(COLOR_GREEN,  "[  ALL TESTS PASSED  ] \n");
```

Comment:
Axxon is not used by Eclipse OMR.  I do like the idea of the test cases printing this sort of thing as it makes it easier to see errors or the lack of errors.  What I am suggesting is that our testMain functions could check the return of RUN_ALL_TESTS and print the `ALL TESTS PASSED` if the return value is 0.  This change does not have to be made under this PR as this is exactly what we were doing in the previous version but I am just curious if you think my idea would work.